### PR TITLE
mysql8: update to 8.4.0

### PR DESCRIPTION
**Pull request by herbygillot**
- mysql82 and mysql81 are now replaced by mysql8

Fixes: https://trac.macports.org/ticket/69934

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with ~`sudo port -vst install`~ `sudo port -d install`?
- [x] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: databases/mysql81/Portfile [hunk 1]
```diff
@@ -1,373 +1,17 @@
...
-"
-
-    livecheck.type          none
+PortSystem          1.0
+PortGroup           obsolete 1.0
+
+name                mysql81
+replaced_by         mysql8
+version             8.1.0
+revision            5
```
Comment (ryandesign) [inline databases/mysql81/Portfile:9]: The revision must be higher than the previous revision for the upgrade to be offered to the user.
```suggestion
revision            6
```

File: databases/mysql81/Portfile [hunk 2]
```diff
@@ -1,373 +1,17 @@
...
+license             {GPL-2 OpenSSLException}
+categories          databases
+
+subport ${name}-server {
+    replaced_by     mysql8-server
+    revision        2
```
Comment (ryandesign) [inline databases/mysql81/Portfile:15]:
```suggestion
    revision        3
```

File: databases/mysql82/Portfile [hunk 3]
```diff
@@ -1,375 +1,17 @@
...
-"
-
-    livecheck.type          none
+PortSystem          1.0
+PortGroup           obsolete 1.0
+
+name                mysql82
+replaced_by         mysql8
+version             8.2.0
+revision            1
```
Comment (ryandesign) [inline databases/mysql82/Portfile:9]:
```suggestion
revision            2
```

File: databases/mysql82/Portfile [hunk 4]
```diff
@@ -1,375 +1,17 @@
...
+license             {GPL-2 OpenSSLException}
+categories          databases
+
+subport ${name}-server {
+    replaced_by     mysql8-server
+    revision        0
```
Comment (ryandesign) [inline databases/mysql82/Portfile:15]:
```suggestion
    revision        1
```

**Comment by herbygillot** [comment]
> The revision must be higher than the previous revision for the upgrade to be offered to the user.
Thanks, addressed.

**Comment by herbygillot** [comment]
Removed the commits that deleted the `mysql82` & `mysql81` variants from Qt5/6 and p5-dbd, will add those as a separate commit/PR.

**Pull request merged**